Client/game_sa/CFxSA.h

```diff
@@ -29,6 +29,7 @@ struct RwColor;
 #define FUNC_CFx_TriggerWaterSplash        0x4a1070
 #define FUNC_CFx_TriggerBulletSplash       0x4a10e0
 #define FUNC_CFx_TriggerFootSplash         0x4a1150
+constexpr auto SHADOW_BASE_TEXTURE_OFFSET = 0xC403E0;
```

Comment:
This points to `struct RwTexture * gpShadowCarTex`, but the other shadow textures are fortunately linearly right after this one.